### PR TITLE
[SDESK-3732] Refactor the Send-to pane to introduce more space for th…

### PR DESCRIPTION
**Pull request by fritzSF**
…e Publish section

### CODE REVIEW

**Comment by fritzSF** [comment]
Might need some code optimisation.

File: scripts/core/ui/views/sd-datepicker.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="datepicker-input">
      <input type="text"
-         class="input-small"
+         class=""
```
Comment (tomaskikutis) [inline scripts/core/ui/views/sd-datepicker.html]: Remove empty attribute

Comment (fritzSF) [reply scripts/core/ui/views/sd-datepicker.html]: UI framework is updated, it should be fine once the PR is merged. Class removed

**Comment by tomaskikutis** [comment]
I've tested again and the UI problems are still there. I don't think anything will change if you merge it to master. The latest release of UI framework is from 20 days ago, that might be the problem.

**Comment by fritzSF** [comment]
Tomas, please check the _Work in progress_ label on this PR, it won't be merged yet. The UI framework is updated 2 days ago but probably not deployed in SD. 

The right side of the Publish pane is that way by design as there will be additional content – mostly Publisher related stuff. We will most likely introduce some rules to hide it if publisher isn't integrated.

**Comment by tomaskikutis** [comment]
Sorry, I didn't see the WIP label. To use the latest version of UI framework, a release has to be made on GitHub and version updated in `package.json` - that was done 20 days ago - so make sure you do that if you want to use a later version.

**Pull request merged**